.editorconfig

```diff
@@ -1,33 +0,0 @@
-# The EditorConfig project consists of a file format for defining coding styles
```

Comment:
@juexinwang : please do not remove this file, this is not an eclipse specific file. This file was initially added to have some coding standards across different editors. see http://editorconfig.org/ for details.